[2024-12-19T05:42:06.948+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T05:42:06.960+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-05-11T00:00:00+00:00 [queued]>
[2024-12-19T05:42:06.964+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-05-11T00:00:00+00:00 [queued]>
[2024-12-19T05:42:06.964+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T05:42:06.972+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2023-05-11 00:00:00+00:00
[2024-12-19T05:42:06.976+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2907) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T05:42:06.978+0000] {standard_task_runner.py:72} INFO - Started process 2912 to run task
[2024-12-19T05:42:06.977+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2023-05-11T00:00:00+00:00', '--job-id', '655', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmppub4kayu']
[2024-12-19T05:42:06.979+0000] {standard_task_runner.py:105} INFO - Job 655: Subtask first_task
[2024-12-19T05:42:07.004+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2023-05-11T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T05:42:07.045+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-11T00:00:00+00:00'
[2024-12-19T05:42:07.046+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T05:42:07.055+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T05:42:07.055+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T05:42:07.060+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T05:42:07.061+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T05:42:07.062+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T05:42:07.074+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T05:42:07.077+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2023-05-11T00:00:00+00:00, execution_date=20230511T000000, start_date=20241219T054206, end_date=20241219T054207
[2024-12-19T05:42:07.118+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T05:42:07.126+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T05:42:07.127+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
